### PR TITLE
Feature/Refactor Fuel Information Related Vehicle Data

### PR DESCRIPTION
**Pull request by Mariia-Novosiadla**
Provides the information for [#3124](https://github.com/smartdevicelink/sdl_core/issues/3124)
- Proposal: [Refactor Fuel Information Related Vehicle Data](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md) + UPD per [pull #939](https://github.com/smartdevicelink/sdl_evolution/pull/939)

### Summary:

1. Adding new enum `CapacityUnit` 
2. Extending `FuelRange` struct by new params: `level` , `levelState`, `capacity`, `capacityUnit`

### CLA
* [x]  I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: docs/Common/Structs/index.md [hunk 1]
```diff
@@ -752,6 +752,10 @@
 |:---|:---|:--------|:---------|:----------|
 |type|[Common.FuelType](../enums/#fueltype)|false|||
 |range|Float|false|minvalue: 0<br>maxvalue: 10000|The estimate range in KM the vehicle can travel based on fuel level and consumption|
+|level|Float|false|minvalue: 0<br>maxvalue: 10000|The relative remaining capacity of this fuel type (percentage)|
```
Comment (LitvinenkoIra) [inline docs/Common/Structs/index.md:755]: minvalue: -6

File: docs/Common/Structs/index.md [hunk 2]
```diff
@@ -752,6 +752,10 @@
...
+|levelState|[Common.ComponentVolumeStatus](../enums/#componentvolumestatus)|false||The fuel level state|
+|capacity|Float|false|minvalue: 0<br>maxvalue: 10000|The absolute capacity of this fuel type|
```
Comment (LitvinenkoIra) [inline docs/Common/Structs/index.md:757]: maxvalue: 1000000

File: docs/Common/Structs/index.md (hunk 1 above)
Comment (Mariia-Novosiadla) [reply docs/Common/Structs/index.md]: Updated in 86be035deb4426cac61bd292dae37c3be8ae1f64

File: docs/Common/Structs/index.md (hunk 2 above)
Comment (Mariia-Novosiadla) [reply docs/Common/Structs/index.md]: updated in 86be035deb4426cac61bd292dae37c3be8ae1f64

**Comment by LitvinenkoIra** [comment]
@atiwari9 please review this PR

**Comment by atiwari9** [comment]
PR Approved.

**Comment by LitvinenkoIra** [comment]
@theresalech PR is ready for Livio review. Thank you!

File: docs/Common/Enums/index.md [hunk 3]
```diff
@@ -1302,3 +1302,10 @@
 |MOBILE|0||
 |CLOUD|1||
 |BOTH|2||
+
+### CapacityUnit
+|Name|Value|Description|
+|:---|:----|:----------|
+|LITERS|0||
+|KILOWATTHOURS|1||
+|KILOGRAMS|2||
```
Comment (ShobhitAd) [inline docs/Common/Enums/index.md:1311]:
```suggestion
### CapacityUnit

|Name|Value|Description|
|:---|:----|:----------|
|LITERS|0||
|KILOWATTHOURS|1||
|KILOGRAMS|2||

```

**Pull request merged**